src/_guides/openapi/specification/v3.1/introduction/benefits.md

```diff
@@ -67,6 +67,7 @@ There are many tools to help you get the most out of OpenAPI, at every step of t
 
 ### Testing
 
+* [Bump.sh](https://bump.sh/api-documentation) 💙
```

Comment:
```suggestion
```

It's not "testing" in the same sense. Testing, in this paragraph, is about automated/technical tests